lib/instrumentation/named-array.js

```diff
@@ -0,0 +1,47 @@
+// TODO: Move to a separate module
```

Comment:
Add `'use strict'` at the top of the file

Reply:
fixed